integration/utils/follower.go

```diff
@@ -128,9 +128,20 @@ Loop:
 		getBlockByHeightDuration := time.Since(GetBlockByHeightTime)
 
 		var blockTxs, blockUnknownTxs uint64
+
+		for _, guaranteed := range block.CollectionGuarantees {
+			_, err := f.client.GetCollection(f.ctx, guaranteed.CollectionID)
+			if err != nil {
+				continue Loop // silently try again if not ready; can happen several dozen times
+			}
+		}
+
 		for _, guaranteed := range block.CollectionGuarantees {
 			col, err := f.client.GetCollection(f.ctx, guaranteed.CollectionID)
 			if err != nil {
+				f.logger.Error().
+					Uint64("blockHeight", block.Height).
+					Msg("GetCollection() error") // should never happen here
 				continue Loop
```

Comment:
This should panic if it should never happen.  This comment may not be relevant.  See above comment.

Reply:
> This should panic if it should never happen

Possibly, but in theory the loader can continue so why not continue? Seeing the frequency of this error and then investigating might be more useful than forcing panic upon the first time the error occurs? Presumably this error can only occur due to some kind of concurrent writing to the data structures, anyway? If this error would only happen for 1 in 10,000 blocks then it hasn't done that much harm in the big scheme of things, but a panic would be annoying? Yes, the panic would force examination of the issue, but there are plenty of `.Error()` calls in the code base and they don't need t be panics, or? IMO the loader should somehow monitor for any errors from any nodes and if too many of them occur then abort anyway since it's not a 'clean' load test anymore?